backend/inspirehep/mailing/providers/mailchimp.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import structlog
+from flask import current_app
+from mailchimp3 import MailChimp
+
+from ..errors import MailChimpMissingAPIToken
+
+LOGGER = structlog.getLogger()
+
+
+def get_mailchimp_client():
+    MAILCHIMP_API_TOKEN = current_app.config.get("MAILCHIMP_API_TOKEN")
+
+    if not MAILCHIMP_API_TOKEN:
+        raise MailChimpMissingAPIToken("Mailchimp API token is not set.")
+
+    return MailChimp(
+        mc_api=MAILCHIMP_API_TOKEN,
+        timeout=current_app.config.get("MAILCHIMP_CLIENT_TIMEOUT"),
+    )
+
+
+def create_mailchimp_campaign(replicate_campaign_id):
+    """Create a mailchimp campaign.
+
+    Args:
+        replicate_campaign_id (str): the campaign id to replicate.
+
+    Retruns:
+        str: the new campaign id.
+    Note:
+        The campaigns can be retrieved by running:
+        > MailChimp(mc_api=MAILCHIMP_API_TOKEN).campaigns.all()
+    """
+    client = get_mailchimp_client()
+    if not client:
+        return
```

Comment:
Is it good idea to return silently here?

Reply:
it's already throwing an exception https://github.com/inspirehep/inspirehep/pull/420/files#diff-a31ae48db9391f7531b5121831fc4833R21 but indeed this check is redundant